components/molecule/dataCounter/src/index.js

```diff
@@ -2,36 +2,43 @@ import {useState} from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
 
-import AtomButton from '@s-ui/react-atom-button'
+import AtomButton, {atomButtonSizes} from '@s-ui/react-atom-button'
 import AtomInput, {inputSizes} from '@s-ui/react-atom-input'
 import MoleculeField from '@s-ui/react-molecule-field'
+import AddIcon from '@adv-ui/fc-iconset/lib/Add'
```

Comment:
Icon might be customizable by consumers of this component. Next line...same comment

Reply:
Fixed 👍